tasks/main.yml

```diff
@@ -132,6 +132,19 @@
   when:
     - deploydrupal_drush_deploy
 
+# Config Structure Check - This is primarily to catch configuration structure
+# changes from updating modules with Dependabot or missed configuration. This
+# works because the exit code for grep is non-zero when the supplied search
+# string is not found.
```

Comment:
```suggestion
# Identify Drupal configuration changes in core or contrib updates that need
# to be exported and may have been missed.
```

Reply:
Good call. Ah, there were remnants of my previous approach in the comment too. 🤦